CRM/Civicase/Hook/PostProcess/SaveCaseCategoryCustomFields.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use CRM_Civicase_Service_CaseCategoryCustomFieldsSetting as CaseCategoryCustomFieldsSetting;
+
+/**
+ * SaveCaseCategoryCustomFields Post Process Hook class.
+ */
+class CRM_Civicase_Hook_PostProcess_SaveCaseCategoryCustomFields extends CRM_Civicase_Hook_CaseCategoryFormHookBase {
+
+  /**
+   * Saves the case category custom field values.
+   *
+   * @param string $formName
+   *   The Form class name.
+   * @param CRM_Core_Form $form
+   *   The Form instance.
+   */
+  public function run($formName, CRM_Core_Form $form) {
+    if (!$this->shouldRun($form, $formName)) {
+      return;
+    }
+
+    $caseCategoryValues = $form->getVar('_submitValues');
+
+    CaseCategoryCustomFieldsSetting::save($caseCategoryValues['value'], [
+      'singular_label' => $caseCategoryValues['case_category_singular_label'],
```

Comment:
Any reason why the names are not same? i.e `singular_label` and `case_category_singular_label`.
Can we try to align both? Will help avoid mistakes if both values are same.

Reply:
Wanted to avoid a generic game because this form is not specific for case categories. But removed the prefix.